### ISSUE TITLE
where is y?

### ISSUE THREAD

**Issue by GuobinZhangTJU** [fursovia/self-adj-dice#2]
Thanks for your sharing.
However, where is the y(i1) in your code compared to the published paper?
probs_with_factor = ((1 - probs) ** self.alpha) * probs
loss = 1 - (2 * probs_with_factor + self.gamma) / (probs_with_factor + 1 + self.gamma)
![image](https://user-images.githubusercontent.com/42600942/94668378-8a6c2480-0342-11eb-8b66-5a51dda47219.png)
I try to use tp + fp + fn to calculate the loss, but it didn't work. 
Here is my code, is there any error?  many thanks.
        pre = (tp + 1e-8) / (tp + fn + 1e-8)
        rec = (tp + 1e-8) / (tp + fp + 1e-8)
        nominator = 2*((1 - pre) ** alpha) * pre * rec + 1
        denominator = ((1 - pre) ** alpha) * pre + rec + 1
        dc = (nominator + 1e-8) / (denominator + 1e-8)
waiting for your reply~

**Comment by fursovia** [comment]
@GuobinZhangTJU The paper provides an example only for the binary case. For multi-class case you don't need `y`. Please take a closer look at the code.
